lib/util/github/graphql/cache-strategies/package-cache-strategy.ts

```diff
@@ -15,23 +15,21 @@ export class GithubGraphqlPackageCacheStrategy<
   async persist(
     cacheRecord: GithubGraphqlCacheRecord<GithubItem>
   ): Promise<void> {
-    if (this.hasUpdatedItems) {
```

Comment:
This caused the problem:
- Unrelated change was introduced here: #21024
- `this.hasUpdatedItems` wasn't set during first fetch
- since data wasn't stored after first fetch, every subsequent fetches became equivalent to the first one